### PR TITLE
Fall Hacks 2024 Website

### PR DESCRIPTION
**Pull request by micahdbak**

### CODE REVIEW

**Comment by micahdbak** [comment]
Content is tentatively approved by Emily - waiting for some final feedback.

Can be seen @ https://new.sfucsss.org/events/2024/fall_hacks/

(Note: we should set up development branch deployments. I'll look into writing a script for this 😉. Could probably just deploy to something like https://new.sfucsss.org/dev/mdb-fall-hacks-2024, maybe)

**Comment by EarthenSky** [comment]
> (Note: we should set up development branch deployments. I'll look into writing a script for this 😉. Could probably just deploy to something like https://new.sfucsss.org/dev/mdb-fall-hacks-2024, maybe)

Could you write a lil bit of docs on how the dev branch works under wiki? Sounds cool!

**Comment by micahdbak** [comment]
> Could you write a lil bit of docs on how the dev branch works under wiki? Sounds cool!

When I get around to implementing this, I'll def write a wiki. ATM though nothing is implemented!

File: public/fall_hacks/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700#display=swap');
+    body {
+      background-color: black;
+    }
+    pre {
+      font-family: 'IBM Plex Mono', monospace;
+      color: lightgreen;
+      font-weight: 400;
+      font-size: 12px;
+      width: 100ch;
```
Comment (EarthenSky) [inline public/fall_hacks/index.html:19]: I had no idea you could do width by characters!

Comment (micahdbak) [reply public/fall_hacks/index.html:19]: It's super cool isn't it!!! Only ever useful with monospace fonts, though

File: src/main/components/Page.js [hunk 2]
```diff
@@ -55,22 +55,25 @@ export const Page = ({ children }) => {
...
       </VSCode.NavFolder>
       <VSCode.NavItem text="login.txt" to="/profile" />
-      <VSCode.NavFolder text="Events">
+      <VSCode.NavFolder text="Hackathons">
         <VSCode.NavLink
           isInFolder={true}
-          text="mtn_madness_23.html"
-          href="https://sfucsss.org/events/mm/2023"
+          text="mtn_madness.html"
+          href="/events/2024/mm"
```
Comment (EarthenSky) [inline src/main/components/Page.js:62]: Should we be consistent with urls and do `/mm/2024` ?

**Review by EarthenSky — APPROVED** [review]
Looks good, just a few questions.

File: src/main/components/Page.js (hunk 2 above)
Comment (micahdbak) [reply src/main/components/Page.js:62]: Yes, 100%, I've made [this issue](https://github.com/CSSS/csss-site-frontend/issues/21) for moving the mountain madness site to be consistent with fall hacks.

**Comment by micahdbak** [comment]
Merging!

**Pull request merged**